src/main/java/core/basesyntax/FruitStorage.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.StorageOperation;
+import core.basesyntax.operations.SupplyAndReturnOperation;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorage {
+    public static Map<String, Integer> fruitStorage;
+    public static Map<String, LocalDate> expiration;
+
+    public Map<String, Integer> createFruitStorage(List<Operation> operations) {
```

Comment:
not sure that you are accepting the operation. This is a list of FruitDto.

Reply:
yes, changed that class to FruitDto